src/NinjaTools.FluentMockServer.API/Proxy/Evaluation/Visitors/IVisitor.cs

```diff
@@ -0,0 +1,11 @@
+namespace NinjaTools.FluentMockServer.API.Proxy.Evaluation.Visitors
+{
+    public interface IRequestMatcherEvaluatorVistor<T> : IRequestMatcherEvaluatorVistor
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'out' keyword to parameter 'T' to make it 'covariant'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)